### DISCUSSION TITLE
【需求】开放自定义markdown解析拓展支持

### DISCUSSION THREAD

**Discussion by ma-x-x** [imzbf/md-editor-v3#227] [Show and tell]
### 需求内容

看代码内部是用Marked 进行markdown内容解析的，marked支持自定义扩展，是否考虑开放用户可添加扩展功能。

### 使用场景

可以根据链接地址中对应的后缀名去做相应的展示，例如展示视频、音频、pdf、json html等。

**Comment by imzbf** [comment]
现在就是支持的，
你可以随意的定义`marked`默认的解析方式，比如标题等。
同时也可以添加`marked`的扩展和一些构建参数。
使用[MdEditor.config](https://imzbf.github.io/md-editor-v3/docs#%F0%9F%92%B4%20Config%20Editor)。

**Comment by ma-x-x** [comment]
您好，我已经收到您的来件，我会尽快处理的。祝您工作生活愉快。                                              ——BastogneYu
